index.js

```diff
@@ -317,9 +322,9 @@ module.exports = function (config) {
     return bddCheck;
   }
 
-  function addScreenshotToReport(step) {
+  function addScreenshotToReport(step, file_path) {
```

Comment:
hmm I wonder instead of adding file_path argument we just change the method to only take filename.
since all we do with the step object is use it to obtain the filename

we can probably just change line 76 from
```
if (step.helperMethod === 'saveScreenshot') addScreenshotToReport(step);
```
to
```
if (step.helperMethod === 'saveScreenshot') {
  const filePath = path.join(global.output_dir, step.args[0]);
  addScreenshotToReport(filePath);
}
```

Reply:
Yeah, that's a good idea.  I'll make the update now.